editor/project_export.cpp

```diff
@@ -1029,6 +1029,7 @@ ProjectExportDialog::ProjectExportDialog() {
 	export_filter->add_item(TTR("Export all resources in the project"));
 	export_filter->add_item(TTR("Export selected scenes (and dependencies)"));
 	export_filter->add_item(TTR("Export selected resources (and dependencies)"));
+	export_filter->add_item(TTR("Exclude selected resources"));
```

Comment:
This description is quite long, but more explicit:

```suggestion
	export_filter->add_item(TTR("Export all resources in the project except resources checked below"));
```